app/helpers/react_components/mentoring/discussion.rb

```diff
@@ -19,6 +19,7 @@ def to_s
             },
             track: {
               title: track.title,
+              slug: track.slug,
```

Comment:
I use the track slug as the `language` parameter for Highlight JS. We might eventually need to do a mapping for this.